src/hardware/tandy_sound.cpp

```diff
@@ -303,22 +303,30 @@ void TandyDAC::ChangeMode()
 		if (regs.clock_divider == 0) {
 			return;
 		}
-		if (const auto sample_rate = tandy_psg_clock_hz / regs.clock_divider;
-		    sample_rate < dac_max_sample_rate_hz) {
+
+		if (const auto new_sample_rate = tandy_psg_clock_hz / regs.clock_divider;
+		    new_sample_rate < dac_max_sample_rate_hz) {
```

Comment:
Should we log something if the new_sample_rate is out-of-bounds?

Reply:
Yeah, I might put an assert there, ok. But I did not want to do deeper refactoring this time around.

Reply:
Actually, we don't want to log out-of-bounds values. See the comment a few lines above this line.

I remember a discussion about this, this would result in hundreds of log entries in some games that set very high sample rates first, which then "stabilises" to something within the normal range.

> 	// Typical sample rates are 1.7. 5.5, 11, and rarely 22 KHz. Although
	// several games (one being OutRun) set instantaneous rates above
	// 100,000 Hz, we throw these out as they can cause garbage high
	// frequency harmonics and also cause problems for the Speex Resampler.
	// For example, a clock divider value of 8 (which is valid) produces a
	// 450 KHz sampling rate, which is way beyond what Speex can handle.
	//